powerlevel9k.zsh-theme

```diff
@@ -744,9 +744,11 @@ prompt_docker_machine() {
 # GO prompt
 prompt_go_version() {
   local go_version
+  local go_path
   go_version=$(go version 2>/dev/null | sed -E "s/.*(go[0-9.]*).*/\1/")
+  go_path=$(go env GOPATH)
```

Comment:
This prints an error message if no `go` is installed. I know one should not activate the go segment, if he does not intend to install go, but I like segments to degrade gracefully.. So a `$(go env GOPATH 2>/dev/null)` would be nice.